src/modules/pulseaudio.cpp

```diff
@@ -222,28 +223,53 @@ auto waybar::modules::Pulseaudio::update() -> void {
       }
       format_name = format_name + "-muted";
       label_.get_style_context()->add_class("muted");
+      label_.get_style_context()->add_class("sink-muted");
```

Comment:
This class is really needed?
Seems a dups of `muted` one

Reply:
This is to separately handle sink and source mutes, while preserving default behavior.